resources/js/components/attribute/Tabular.vue

```diff
@@ -620,8 +510,8 @@
                 initialValue: value.value,
             });
             const state = reactive({
-                isPreview: computed(_ => previewColumns.value && Object.keys(previewColumns.value).length > 0),
-                columns: computed(_ => state.isPreview ? previewColumns.value : getAttribute(attribute.value.id).columns),
+                isPreview: computed(_ => preview.value && previewData.value && Object.keys(previewData.value).length > 0),
```

Comment:
Why isn't preview.value check enough?
Maybe a different name if it does more than one thing?

Reply:
I just did a double check to make sure that we enabled preview (`preview.value`) and have actual data to preview (`previewData.value`).

Reply:
I think there was a mix up from the old structure. `preview` and `previewData` is part of `Attribute.vue` and `Tabular.vue` only receives `previewColumns`. Thus, checking for `previewColumns` existance and content is enough.